e2e_deployment_files/nestededge_bottomLayerBaseDeployment_connectivity_mqtt.template.json

```diff
@@ -680,6 +701,21 @@
                 }
               ]
             }
+          ],
+          "bridges": [
+            {
+              "endpoint": "$upstream",
+              "settings": [
+                {
+                  "direction": "in",
+                  "topic": "relay/#"
+                },
+                {
+                  "direction": "out",
+                  "topic": "initiate/#"
+                }
+              ]
+            }
```

Comment:
The changes in this file are due to a missing cherry pick. I wanted to test the behavior for this module as it is enabled in 1.2. I will revert before merge, just using it for testing.